packages/typespec-client-generator-core/test/types/built-in-types.test.ts

```diff
@@ -134,15 +138,9 @@ describe("typespec-client-generator-core: built-in types", () => {
       @access(Access.public)
       model Test {
         urlScalar: url;
-        uuidScalar: uuid;
-        eTagScalar: eTag;
 
         @format("url")
         urlProperty: string;
-        @format("uuid")
-        uuidProperty: string;
-        @format("eTag")
-        eTagProperty: string;
```

Comment:
these are azure tpyes, we should not support their format any more

Reply:
Why do we remove below cases? 
```
uuidScalar: uuid;
eTagScalar: eTag;
```